Makefile

```diff
@@ -179,6 +179,10 @@ lint:
 unit:
 	hack/unit.sh ${GO_MOD_FLAGS}
 
+.PHONY: community
+community:
```

Comment:
It seems like the release PR relies on the introduction of this make target, but the changes to common.sh are assuming the release PR will go in first.

Reply:
What order should they go in without breaking each other?

Reply:
I thought about this more - I am thinking of dropping the common.sh commit from this PR and add it to a follow-up PR after the release PR goes in. PR ordering:
1. WMCO PR: add make target and changes to generate.sh
2. release PR: add community testing to the aws job
3. followup WMCO PR: changes to common.sh